src/if_perl.xs

```diff
@@ -40,7 +40,7 @@
 /* Work around for perl-5.18.
  * Don't include "perl\lib\CORE\inline.h" for now,
  * include it after Perl_sv_free2 is defined. */
-#ifdef DYNAMIC_PERL
+#if (PERL_REVISION == 5) && (PERL_VERSION == 18)
```

Comment:
This comment says "work around for perl-5.18", but I think actually this is a workaround for perl-5.18 and later.
Not sure which versions are relevant, though.

Reply:
done: https://github.com/vim/vim/pull/12575/commits/6abf8cdab0bf03ea05cec19d538d4387d730a933